content/accounts/manage-account.md

```diff
@@ -0,0 +1,68 @@
+---
+description: Learn how to manage settings for your Docker account.
+keywords: accounts, docker ID, account settings, account management, docker home
+title: Manage your Docker account
+---
+
+You can manage the settings for your Docker account in Docker Home. Here you can also take administrative actions for your account and manage your account security.
+
+> **Tip**
+>
+> If your account is associated with an organization that enforces Single Sign-On (SSO), you may not have permissions to update your account settings. You need to contact your administrator to update your settings.
+{ .tip }
+
+## Update general settings
+
+1. Sign in to your [Docker account](https://account.docker.com/login).
+2. In Docker Home, select your avatar in the top-right corner to open the dropdown.
+3. Select **My Account**.
```

Comment:
I think this would be `Account Settings`, not `My Account`.

Reply:
Will that be changing soon? I still see **My Account** in stage.

Reply:
Maybe it is `My Account`. Sorry!

Reply:
Maybe not for long 😅